### ISSUE TITLE
Getter for attributedString == crash

### ISSUE THREAD

**Issue by Davarg** [stephsharp/MaterialTextField#11]
If you try to get attributedString that exist but empty you will catch app crash

```objective-c
- (NSAttributedString *)attributedString:(NSAttributedString *)attributedString withColor:(UIColor *)color
{
    NSMutableDictionary *attributes = [[attributedString attributesAtIndex:0 effectiveRange:NULL] mutableCopy];
    attributes[NSForegroundColorAttributeName] = color;

    return [[NSAttributedString alloc] initWithString:attributedString.string attributes:attributes];
}
```

The problem is in this line of code
```objective-c
NSMutableDictionary *attributes = [[attributedString attributesAtIndex:0 effectiveRange:NULL] mutableCopy];
```

**Comment by stephsharp** [comment]
Thanks @Davarg, here's a fix:

``` objective-c
- (NSAttributedString *)attributedString:(NSAttributedString *)attributedString withColor:(UIColor *)color
{
    NSMutableDictionary *attributes;

    if (attributedString.length > 0) {
        attributes = [[attributedString attributesAtIndex:0 effectiveRange:NULL] mutableCopy];
    }
    else {
        attributes = [NSMutableDictionary dictionary];
    }
    attributes[NSForegroundColorAttributeName] = color;

    return [[NSAttributedString alloc] initWithString:attributedString.string attributes:attributes];
}
```

I'm working on some other iOS 11 updates, so I'll push this out soon. Thanks!

**Comment by stephsharp** [comment]
Fixed in v0.2.1, see 803e7cbd9229c6f35823c8b2d3b263658cf50433
